From 18c73075c3b3503080001b8a8449f387e2402275 Mon Sep 17 00:00:00 2001
From: Bai Ping <ping.bai@nxp.com>
Date: Fri, 2 Nov 2018 14:28:27 +0800
Subject: [PATCH 4994/5242] MLK-20175 ARM64: dts: freescale: Correct the pmu
 interrupt parent of i.mx8mq/mm

commit  37f9a3fdb489634d04b802dd778b27bc8e1c846e from
https://source.codeaurora.org/external/imx/linux-imx.git

The PMU's interrupt is PPI. Correct the pmu interrupt parent of i.MX8MQ/MM.

Signed-off-by: Bai Ping <ping.bai@nxp.com>
Reviewed-by: Anson Huang <Anson.Huang@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8mm.dtsi |    4 ++++
 arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi |    4 ++++
 2 files changed, 8 insertions(+)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8mm.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8mm.dtsi
index 8dd105e..285775a 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8mm.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8mm.dtsi
@@ -111,6 +111,10 @@
 		interrupt-parent = <&gic>;
 	};
 
+	pmu {
+		interrupt-parent = <&gic>;
+	};
+
 	busfreq { /* BUSFREQ */
 		compatible = "fsl,imx_busfreq";
 		clocks = <&clk IMX8MM_DRAM_PLL_OUT>, <&clk IMX8MM_CLK_DRAM_ALT_SRC>,
diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi
index 35dbb26..eec6bbb 100755
--- a/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi
@@ -112,6 +112,10 @@
 		arm,no-tick-in-suspend;
 	};
 
+	pmu {
+		interrupt-parent = <&gic>;
+	};
+
 	busfreq { /* BUSFREQ */
 		compatible = "fsl,imx_busfreq";
 		clocks = <&clk IMX8MQ_DRAM_PLL_OUT>, <&clk IMX8MQ_CLK_DRAM_ALT>,
-- 
1.7.9.5

